interfaces/apparmor/template.go

```diff
@@ -586,6 +587,9 @@ profile snap-update-ns.###SNAP_NAME### (attach_disconnected) {
   #   /var/snap/*/**
   # is meant to mean:
   #   /var/snap/$SNAP_NAME/$SNAP_REVISION/
+  # NOTE: the wildcards below are there because content interface can do content
+  # sharing from one snap to another and we just don't know yet. Those should go
+  # away when the content interface starts using AddUpdateNS() snippets.
```

Comment:
Should that happen in this PR? If not, can you turn this into a FIXME or TODO?

Reply:
It will mostly happen in the next PR